cmd/main.go

```diff
@@ -25,7 +25,7 @@ var (
 	loginPath           = kingpin.Flag("login-path", "Vault path to authenticate against").Required().String()
 	authRole            = kingpin.Flag("auth-role", "Kubernetes authentication role").Required().String()
 	secretPath          = kingpin.Flag("secret-path", "Path to secret in Vault. eg. database/creds/foo").Required().String()
-	caCert              = kingpin.Flag("ca-cert", "Path to CA certificate to validate Vault server").String()
+	caCert              = kingpin.Flag("ca-cert", "Path to CA certificate/certificate folder to validate Vault server").String()
```

Comment:
Can you make it required, so that it gives a clearer error than exploding at os.Stat?

Reply:
I've made it check if ca cert is set instead as there may be valid reasons not to use this flag